### PR TITLE
Create plugin for Redis sink connector

### LINKED ISSUE #288: Add Redis sink connector
create a new extractor plugin for Redis sink connector. This is needed to demo the new kpops word count pipeline.

### PR DESCRIPTION
**Pull request by disrupted**
Closes #288

### CODE REVIEW

**Review by MichaelKora — DISMISSED** [review]

**Comment by disrupted** [comment]
I found that when using multiple input topics, the naming is inconsistent for the sink. will have to revert it back to using the Redis host name instead of input topic name for the database sink

File: backend/streams_explorer/core/extractor/default/redis_sink.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            database = f"{config['redis.hosts']}-db-{config['redis.database']}"
+            self.sinks.append(
+                Sink(
+                    node_type="database",
+                    name=database,
+                    source=connector_name,
+                )
+            )
+            return RedisSinkConnector(
+                type=KafkaConnectorTypesEnum.SINK,
```
Comment (philipp94831) [inline backend/streams_explorer/core/extractor/default/redis_sink.py:32]: shouldn't this be part of the class itself?

File: backend/streams_explorer/core/extractor/default/redis_sink.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+                name=connector_name,
+                config=config,
+                topics=topics,
+                error_topic=None,
```
Comment (philipp94831) [inline backend/streams_explorer/core/extractor/default/redis_sink.py:36]: Shouldn't this be set in the same way as for ES sink?

**Review by philipp94831 — CHANGES_REQUESTED** [review]

File: backend/streams_explorer/core/extractor/default/redis_sink.py (hunk 1 above)
Comment (disrupted) [reply backend/streams_explorer/core/extractor/default/redis_sink.py:32]: yes, that's right https://github.com/bakdata/streams-explorer/issues/223

File: backend/streams_explorer/core/extractor/default/redis_sink.py (hunk 2 above)
Comment (disrupted) [reply backend/streams_explorer/core/extractor/default/redis_sink.py:36]: yeah, I think we can remove this argument as it's the default anyways and rely on the default implementation to scan for `errors.deadletterqueue.topic.name`. is that what you mean?

File: backend/streams_explorer/core/extractor/default/redis_sink.py (hunk 1 above)
Comment (philipp94831) [reply backend/streams_explorer/core/extractor/default/redis_sink.py:31]: But can't you set it in the init method of RedisSinkConnector?

Comment (disrupted) [reply backend/streams_explorer/core/extractor/default/redis_sink.py]: good point!

**Review by philipp94831 — APPROVED** [review]

**Pull request merged**